en_US/mqtt-over-quic/introduction.md

```diff
@@ -1,68 +1,149 @@
-- **Streams with different QoS:** For example, in "Flow Control", QoS 0 messages should give way to high QoS messages.
-- **Separate control messages into different streams:** MQTT control messages can be sent in one or two directions. For example, the client can send UNSUBSCRIBE requests asynchronously through a short-lived unidirectional stream to request the server to stop sending data that is no longer of interest.
-- **Finer-grained send and receive collaborative flow control:** Flow control is performed on a per-flow basis or across the entire connection, enabling finer-grained flow control.
 
 ## QUIC vs TCP/TLS Test Comparisons
 
-In comparison with TCP/TLS testing, MQTT over QUIC performs as follows:
+In comparison with TCP/TLS testing, the performance of MQTT over QUIC is summarized as follows:
 
-1. When network latency is high, QUIC is able to establish connections and subscribe faster.
-2. After disconnecting, it takes less latency to re-establish the connection and reconnect to QUIC.
-3. QUIC is better than TLS for both server CPU and memory usage when connecting/reconnecting on a large scale.
-4. When NAT rebinding, client reconnection response under TCP/TLS is very slow and the message transmission is broken, while QUIC handles it more smoothly and the messages are sent without any impact.
-5. In a weak network packet loss and packet transmission disorder environment, TLS shows message congestion and loss due to a poor network environment, while the QUIC server receives slightly jittery data but does not lose messages.
+- When network latency is high, QUIC is able to establish connections and subscribe faster.
+- After disconnecting, with 0-RTT, it could re-establish the connection much quicker than TCP/TLS.
+- QUIC is better than TLS for both server CPU and memory usage when connecting/reconnecting on a large scale.
+- When NAT rebinding, client reconnection response under TCP/TLS is very slow and the message transmission is broken, while QUIC handles it more smoothly and the messages are sent without any impact.
+- In a weak network packet loss and packet transmission disorder environment, TLS shows message congestion and loss due to a poor network environment, while the QUIC server receives slightly jittery data but does not lose messages.
 
 ## Future Work
 
 As of now, the MQTT over QUIC is ready for production, users are already testing it in depth and giving good feedback, see [Getting Started](./getting-started.md) to experience it now.
 
-Still, EMQX has not utilized all the features provided by QUIC, such as multi streams, stream prioritization, flow control and unreliable datagram.
+Still, EMQX has not utilized all the features provided by QUIC, such as broker-side stream prioritization, broker-side flow control, and unreliable datagram.
 
```

Comment:
Add more stuff as following,

Further investigation is required to determine how to handle the state of QoS1 and QoS2 messages in case of an abnormal closure of the data stream including the subscription resumption without reconnect.